packages/flutter/lib/src/material/about.dart

```diff
@@ -1052,9 +1052,9 @@ class _MasterDetailFlow extends StatefulWidget {
   const _MasterDetailFlow({
     required this.detailPageBuilder,
     required this.masterViewBuilder,
-    this.automaticallyImplyLeading = true, // ignore: unused_element
+    this.automaticallyImplyLeading = true, // ignore: unused_element, unused_element_parameter
     this.detailPageFABlessGutterWidth,
-    this.displayMode = _LayoutMode.auto, // ignore: unused_element
+    this.displayMode = _LayoutMode.auto, // ignore: unused_element, unused_element_parameter
```

Comment:
Unrelated to your PR: We should probably just remove these parameters on _MasterDetailFlow.

Reply:
I'd be OK with that too 😁

Reply:
https://github.com/flutter/flutter/pull/126935